topostats/default_config.yaml

```diff
@@ -54,9 +58,9 @@ plotting:
   run: true # Options : true, false
   save_format: png # Options : see https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.savefig.html
   pixel_interpolation: null # Options : https://matplotlib.org/stable/gallery/images_contours_and_fields/interpolation_methods.html
-  image_set: core  # Options : all, core
-  zrange: [null, null]  # low and high height range for core images (can take [null, null]). low <= high
-  colorbar: true  # Options : true, false
+  image_set: all # Options : all, core
```

Comment:
Will go through in more detail over the coming days but I think the desired default here should be `core` as this file serves for output from `--create-config-file` and I thought most of the time people don't require all grain images.

Reply:
This change was pushed in error, I was testing output images and forgot to change this back, thanks!